use-cases/inferencing/cost-optimization/gcsfuse/README.md

```diff
@@ -0,0 +1,271 @@
+
+  ```sh
+  watch --color --interval 5 --no-title \
+  "kubectl --namespace ${MLP_MODEL_SERVE_NAMESPACE} get deployment/vllm-openai-gcs-llama33-70b-${ACCELERATOR} | GREP_COLORS='mt=01;92' egrep --color=always -e '^' -e '1/1     1            1'"
+  ```
+
+  It can take 20+ minutes for the deployment to be ready and available.
+
+  ```
+  NAME                              READY   UP-TO-DATE   AVAILABLE    AGE
+  vllm-openai-gcs-llama33-70b-a100   1/1     1            1           XXXXX
+  ```
+
+## Calculate pod startup time
+
+- Install bc in CloudShell
+
+  ```sh
+  sudo apt-get install bc
+  ```
+
+- Run the following commands to get the model load time in seconds
+
+  ```sh
+  POD_NAME=`kubectl --namespace "${MLP_MODEL_SERVE_NAMESPACE}" get pods -o json | jq -r --arg accelerator "$ACCELERATOR" ".items[] | select(.metadata.name | contains(\"vllm-openai-gcs-llama33-70b-$accelerator\")) | .metadata.name"`
+
+  BEGIN_MODEL_DOWNLOAD_TIME=`kubectl --namespace ${MLP_MODEL_SERVE_NAMESPACE} logs ${POD_NAME} -c "inference-server" | grep "^INFO.*Starting to load model"  | head -n 1 | awk '{print $2" "$3}' | xargs -I {} date -d "$(date +%Y)-{}" +%s%3N`
+
+  ENDING_MODEL_DOWNLOAD_TIME=`kubectl --namespace ${MLP_MODEL_SERVE_NAMESPACE} logs ${POD_NAME} -c "inference-server" | grep "^INFO.*Loading model weights took" | head -n 1 | awk '{print $2" "$3}' | xargs -I {} date -d "$(date +%Y)-{}" +%s%3N`
+
+  MODEL_LOAD_TIME_WITHOUT_TUNING=`echo "($ENDING_MODEL_DOWNLOAD_TIME - $BEGIN_MODEL_DOWNLOAD_TIME)/1000" | bc`
+
+  echo "MODEL LOAD TIME WITHOUT TUNING - ${MODEL_LOAD_TIME_WITHOUT_TUNING}"
+  ```
+
+  > Loading llama 70B model with GCSFuse and no tuning will typically take
+  > around 1200 seconds.
+
+- Run the following commands to get the pod startup time in seconds. This is the
+  time taken by the pod to change from `PodScheduled` to `Ready` status.
+
+  ```sh
+  POD_SCHEDULED_TIME=`kubectl --namespace "${MLP_MODEL_SERVE_NAMESPACE}" get pods "$POD_NAME" -o json | jq -r '.status.conditions[] | select(.type == "PodScheduled") | .lastTransitionTime' | date -f - +%s%3N`
+
+  POD_READY_TIME=`kubectl --namespace "${MLP_MODEL_SERVE_NAMESPACE}" get pods "$POD_NAME" -o json | jq -r '.status.conditions[] | select(.type == "Ready") | .lastTransitionTime' | date -f - +%s%3N`
+
+  POD_STARTUP_TIME_WITHOUT_TUNING=`echo "($POD_READY_TIME - $POD_SCHEDULED_TIME)/1000" | bc`
+
+  echo "POD STARTUP TIME WITHOUT TUNING - ${POD_STARTUP_TIME_WITHOUT_TUNING}s"
+  ```
+
+## Serve the model with vLLM with tuning
+
+- Configure the deployment.
+
+  ```sh
+  git restore manifests/model-deployment-tuned-${ACCELERATOR}-dws.yaml
+  sed \
+  -i -e "s|V_MODEL_BUCKET|${MLP_STORAGE_BENCHMARK_HIERARCHICAL_BUCKET}|" \
+  -i -e "s|V_MODEL_NAME|${MODEL_NAME}|" \
+  -i -e "s|V_MODEL_VERSION|${MODEL_VERSION}|" \
+  -i -e "s|V_IMAGE_NAME|${VLLM_IMAGE_NAME}|" \
+  -i -e "s|V_KSA|${MLP_STORAGE_BENCHMARKING_KSA}|" \
+  manifests/model-deployment-tuned-${ACCELERATOR}-dws.yaml
```

Comment:
Switch this to use `envsubst` similar to the [RAG guides](https://github.com/GoogleCloudPlatform/accelerated-platforms/blob/main/use-cases/rag-pipeline/embedding-models/multimodal-embedding/README.md)